README.md

```diff
@@ -21,6 +21,8 @@ To bind the java agent to a specific IP change the port number to `host:port`.
 
 See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
 
+Please note that the `/metrics` endpoint is expected to be slow (~10 seconds) and might exceed Prometheus default scrape timeout of 10 seconds.
```

Comment:
That really depends on what you're monitoring

Reply:
Okay. My interpretation of your reply to #175 was that ~10 seconds are expected. Would you be okay with it when I just remove `(~10 seconds)`?

Reply:
10+ seconds is expected for Cassandra, but other things can be fast. I'd word it as a possibility rather than a certainty, and indicate that this is due to the nature of JMX.

Reply:
What about:
> Please note that due to the nature of JMX the `/metrics` endpoint might exceed Prometheus default scrape timeout of 10 seconds.

Reply:
Sounds perfect